caffe2/python/convnet_benchmarks.py

```diff
@@ -563,45 +565,81 @@ def Inception(order, cudnn_ws, model_path=""):
     model.net.AveragedLoss(xent, "loss")
     return model, 224
 
-def Resnet50(order, cudnn_ws, model_path=""):
-    if model_path == "":
-        print("ERROR: please specify paths to init_net and predict_net protobufs for Resnet50")
-        exit(1)
-    device_opts = caffe2_pb2.DeviceOption()
-    device_opts.device_type = caffe2_pb2.HIP
-    device_opts.hip_gpu_id = 0
 
-    INIT_NET_PB = os.path.join(model_path, "init_net.pb")
-    PREDICT_NET_PB = os.path.join(model_path, "predict_net.pb")
-    init_def = caffe2_pb2.NetDef()
-    with open(INIT_NET_PB, 'rb') as f:
-        init_def.ParseFromString(f.read())
-        init_def.device_option.CopyFrom(device_opts)
+def Resnet50(args):
+    gpus = [0]
+    device_opt = core.DeviceOption(caffe2_pb2.HIP)
+    device_opt.hip_gpu_id = gpus[0]
+
+    # Batch size of 32 sums up to roughly 5GB of memory per device
+    batch_per_device = args.batch_size
+    total_batch_size = batch_per_device * len(gpus)
```

Comment:
This is not necessary. It is a single GPU test. batch size and total batch size will always be the same